webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -175,6 +176,36 @@ def test_if_invalid_and_valid_and_second_valid_are_in_prev_step_then_set_prev_ur
             self.assertEqual(previous_url, step.render_info.prev_url)
 
 
+class TestEligibilityStartDisplaySteuerlotseStep(unittest.TestCase):
+    def test_sets_correct_session_data_to_empty_dict(self):
+        session_data = {
+            _ELIGIBILITY_DATA_KEY: create_session_form_data({'marital_status_eligibility': 'single'})
+        }
+        with app.app_context() and app.test_request_context(method='GET') as req:
+            req.session = SecureCookieSession(session_data)
+            step = EligibilityStepChooser('eligibility').get_correct_step(
+                EligibilityStartDisplaySteuerlotseStep.name)
+            step.handle()
+
+            self.assertEqual({}, deserialize_session_data(req.session[_ELIGIBILITY_DATA_KEY]))
+
+    def test_does_not_change_other_session_data(self):
+        other_session_key = 'OTHER_SESSION_KEY'
+        other_session_data = {'Galileo': 'Figaro - magnificoo'}
+        session_data = {
+            _ELIGIBILITY_DATA_KEY: create_session_form_data({'marital_status_eligibility': 'single'}),
+            other_session_key: create_session_form_data(other_session_data)
+        }
```

Comment:
```suggestion
        other_session_key = 'OTHER_SESSION_KEY'
        other_session_data ={ 'OTHER_SESSION_KEY': {'Galileo': 'Figaro - magnificoo'}, 'ANOTHER_SESSION_KEY':  {'Scaramouch': 'Fandango'}}
        session_data = {_ELIGIBILITY_DATA_KEY: create_session_form_data({'marital_status_eligibility': 'single'})} | other_session_data
```
Just a suggestion to ensure that more than one other key is kept. And then we could just assert that the data is other_session_data below.

Reply:
Unfortunately, this way, the other session data is not encrypted. But I'll add an additional key.